spec/support/matchers/be_valid_verbose.rb

```diff
@@ -0,0 +1,17 @@
+RSpec::Matchers.define :be_valid_verbose do
+  match do |model|
```

Comment:
I think it means replace that whole block with `match(&:valid?)` - I'm guessing `model` is the only thing inside the `match` object so you can use the `&:` shorthand to call `valid?` on it without any explicit unwrapping. *makes a personal note of the shorthand for future use*